guides/doc-Release_Notes/topics/katello.adoc

```diff
@@ -4,12 +4,58 @@
 [id="katello-headline-features"]
 == Headline Features
 
+=== Image-mode host support
+
+Katello has added a new type of host: image-mode hosts.
+Image-mode hosts are standard hosts configured to run a bootc bootable container image at startup.
+Katello 4.16 allows users to assign bootable container images to an image-mode host to easily deploy identical system configurations, from basic to complex.
+
+Existing hosts are now considered 'package-mode hosts' and will exist alongside the new host type.
+Both host types can be set up and configured using existing host management processes.
+
+Bootable container images (bootc images) can be managed similarly to existing Foreman repositories.
+A new 'Booted container images' page displays all bootc images and their respective host counts.
+These images are available via API at '/hosts/bootc_images'.
+
+
+=== Flatpak content support
+
+Katello now supports Flatpak content.
+Flatpak content can be manually added as a local repository or can be imported from external sources as a Flatpak remote repository.
+After import, Flatpak remote repositories can be mirrored to local Katello repositories via remote repository mirroring.
+Flatpak repositories (local or remote) can be easily installed and synced on remote hosts via any of Katello's existing repository management techniques.
```

Comment:
```suggestion
Flatpak repositories can be easily synced and installed on remote hosts via any of Katello's existing repository management techniques.
```
I would omit the `(local or remote)` here, it's a bit confusing. During remote repository mirroring a local repository is created and only that can be synced and consumed at a host, in this order.